### ISSUE TITLE
Bandwidth Speed Limit Feature Request!

### ISSUE THREAD

**Issue by imranext** [NidukaAkalanka/x-ui-english#173]
### Welcome

- [X] Yes, I'm using the latest major release. Only such installations are supported.
- [X] Yes, I'm using the supported system. Only such systems are supported.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, log, etc).

### Description of the problem,screencshot would be good

Hello there,
Can you add a feature or a option so that we can limit certain bandwidth speed for client. Example: 40 Mbps for Client A, 60 Mbps for Client B. 

Also can you add a dark mode if possible. Thanks ♥️🚀

### Version of x-ui

<details>

```console
$ x-ui version
# Paste output here
```

</details>

### x-ui log or xray log

<details>

```console
# paste log here
```

</details>

**Comment by drno88** [comment]
yes I support. this is a much needed feature. or tell me how to use it using iptables for example...

**Comment by Mdrzxsony** [comment]
@drno88 See, we use 1G port per server for 10 users and I want to limit each user to 100m/b.
Also, how many users do you think a server with 1G/b port can support and what speed is suitable for the user?

**Comment by beiaduo** [comment]
I need it too
